pkg/lib/operationclient.go

```diff
@@ -1700,9 +1747,11 @@ func (oc *OperationClient) EnableHistogram(bdevName string) error {
 }
 
 func NewOperationClient(sc *SpdkClient) *OperationClient {
-	return &OperationClient{
+	oc := &OperationClient{
 		sc:         sc,
 		nameToBdev: make(map[string]*bdevConf),
 		nqnToNvmf:  make(map[string]*nvmfConf),
 	}
+	oc.LoadBdevs()
```

Comment:
Do not  invoke LoadBdevs in the OperationClient init funciton. The LoadBdevs may be invoked in different scenarios. E.g. in dn Syncup and cn Syncup, it should be invoked after the resource creating stage and before the resource deleting stage. In the GetMetrics GRPC, it should be invoked at start time. So we'd better let the upper layer invoke it explicitly.